daffodil-core/src/main/scala/org/apache/daffodil/core/dsom/DFDLStatementMixin.scala

```diff
@@ -217,7 +217,7 @@ trait ProvidesDFDLStatementMixin extends ThrowsSDE with HasTermCheck {
   final lazy val patternStatements: Seq[DFDLStatement] = patternAsserts ++ patternDiscrims
 
   final lazy val lowPriorityStatements: Seq[DFDLStatement] =
```

Comment:
Any idea where "low priority" comes from? I'm not familiar with that as a DFDL term. Wondering if something like "expressionStatements" would be more clear?

Reply:
Changed to expressionStatements